Example/Okta/ViewController.swift

```diff
@@ -67,16 +68,21 @@ class ViewController: UIViewController {
 
     @IBAction func introspectButton(_ sender: Any) {
         // Get current accessToken
-        guard let accessToken = tokens?.accessToken else { return }
+        guard let accessToken = tokenManager?.accessToken else { return }
```

Comment:
Do we have a task to make this a non-optional property?

Reply:
I don't think that this property could be non-optional. Since before user signs in, we don't have tokenManager available.